vcenter_operator/templates.py

```diff
@@ -182,22 +203,24 @@ def poll(self):
         # self.resource_version = resp['metadata']['resourceVersion']
         for item in resp['items']:
             try:
-                version = item['metadata']['resourceVersion']
-                name = item['metadata']['name']
+                metadata = item['metadata']
+                version = metadata['resourceVersion']
+                name = metadata['name']
                 # This, however, does seem to work
                 # self.resource_version = max(
                 #    version,
                 #    self.resource_version)
-                namespace = item['metadata']['namespace']
+                namespace = metadata['namespace']
```

Comment:
This looks a bit like just cleanup. I'd put that in a different commit, but it's not too bad here. I just got confused because `_owner_from_obj()` does something similar so it seemed related.

Reply:
Moved it to its own commit